pages/linux/snapper.md

```diff
@@ -0,0 +1,27 @@
+# snapper
+
+> Filesystem snapshot management tool.
+
+- List snapshot configs:
+
+`snapper list-configs`
+
+- Create snapper config:
+
+`snapper -c {{config}} create-config {{path/to/subvolume}}`
+
+- List snapshots for a config:
+
+`snapper -c {{config}} list`
+
+- Create a new snapshot:
+
+`snapper -c {{config}} snapshot`
+
+- Delete a snapshot:
+
+`snapper -c {{config}} delete {{snapshot_number}}`
+
+- Delete multiple snapshots:
+
+`snapper -c {{config}} delete {{snapshot_1}}-{{snapshot_2}}`
```

Comment:
For more than 2 snapshots, do you put `-` in between all of them ?

Reply:
I think this may be a better example of this instead,
```
-- Delete multiple snapshots:
+- Delete a range of snapshots:
-`snapper -c {{config}} delete {{snapshot_1}}-{{snapshot_2}}`
+`snapper -c {{config}} delete {{snapshot_X}}-{{snapshot_Y}}`
```
A real world example being `snapper -c home delete 101-120`